ci/platforms/config.gaea

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+export GFS_CI_ROOT=/gpfs/f5/epic/proj-shared/global/GFS_CI_ROOT
+export ICSDIR_ROOT=/gpfs/f5/epic/proj-shared/global/glopara/data/ICSDIR
+export STMP="/gpfs/f5/epic/scratch/${USER}"
+export SLURM_ACCOUNT=epic
+export max_concurrent_cases=5
+export max_concurrent_pr=4
```

Comment:
Is the global-workflow CI running on Gaea?  If not, please add this in the PR that turns the CI ON on Gaea.

Reply:
@aerorahul @AnilKumar-NOAA Anil, you can remove this file..I have a branch for CI on Gaea that will include this.